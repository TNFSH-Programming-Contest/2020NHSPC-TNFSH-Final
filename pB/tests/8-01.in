85
119.003 -215.189
135.603 -339.052
87.08 -333.945
16.848 -171.773
103.68 -154.0
173.912 -198.589
212.22 -129.634
148.373 -44.079
71.756 -74.725
20.679 -90.049
-44.445 -0.663
88.357 64.461
147.096 3.168
204.558 19.768
184.127 77.231
123.2 72.2
101.126 95.108
69.202 132.139
29.617 72.123
-8.691 162.786
-41.892 92.554
-115.954 156.401
-115.954 0.614
-59.769 -95.156
-75.092 -144.957
-132.554 -56.848
-182.355 38.922
-164.478 157.678
-92.969 195.986
-68.707 157.678
-59.769 203.648
-145.324 207.479
-275.572 171.724
-285.787 35.092
-257.695 -63.233
-362.404 -63.233
-361.127 165.34
-417.312 165.34
-368.789 -226.681
-266.633 -217.743
-257.695 -275.205
-404.543 -281.59
-436.467 -187.096
-465.836 -415.669
-410.928 -496.116
-349.635 -491.008
-352.188 -325.006
-207.894 -322.452
-201.509 -217.743
-274.295 -160.28
-302.388 -105.372
-211.725 -113.034
-233.433 -10.878
-183.632 -37.694
-146.601 -138.572
-50.83 -217.743
6.632 -224.128
16.848 -245.836
-54.661 -254.774
-58.492 -373.53
-215.556 -365.868
-215.556 -410.561
-274.295 -410.561
-278.126 -344.16
-312.603 -354.376
-306.219 -461.639
-219.386 -457.808
-135.108 -405.453
-98.077 -420.777
-161.924 -471.854
-207.894 -512.716
-103.0 -496.116
-73.815 -466.747
-34.23 -492.285
-17.63 -307.129
83.249 -430.992
73.033 -499.947
139.434 -505.055
127.942 -438.654
222.435 -489.731
232.651 -420.777
167.527 -390.13
180.296 -317.344
221.158 -353.099
221.158 -280.313
-27.6 -78.6
