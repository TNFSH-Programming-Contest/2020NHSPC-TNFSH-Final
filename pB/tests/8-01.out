1009267.949
